opt_out_http_api/api_methods.py

```diff
@@ -29,10 +33,18 @@ def get_opt_out(self, addresstype, address):
             return opt_outs[0]
         return None
 
-    @app.handle_errors(OptOutNotFound)
-    def opt_out_not_found(self, request, failure):
-        return self.response(
-            request, status_code=404, status_reason="Opt out not found.")
+# Save Opt Out Address
+    # print
+
+    def save_opt_out(self, addresstype, address):
+        x = uuid.UUID('{00010203-0405-0607-0809-0a0b0c0d0e0f}')
```

Comment:
We need `save_opt_out` to create a new UUID each time, not the same one every time.